_client/tests/unit/testMainWindow.cpp

```diff
@@ -0,0 +1,34 @@
+#include <gtest/gtest.h>
+
+#include <QApplication>
+
+#define SPDLOG_ACTIVE_LEVEL SPDLOG_LEVEL_TRACE
```

Comment:
It's not needed, cos we do that in `SpdlogWrapper`

Reply:
deleted